0.10.4/alpine/Dockerfile

```diff
@@ -0,0 +1,77 @@
+FROM alpine:latest
```

Comment:
This file should be on the root, since we're only maintaining one version for now.